tests/platform_tests/mellanox/mellanox_thermal_control_test_helper.py

```diff
@@ -384,7 +403,7 @@ class FanData:
     PWM_MAX = 255
 
     # Speed tolerance
-    SPEED_TOLERANCE = 0.2
+    SPEED_TOLERANCE = 0.5
```

Comment:
Why increase speed tolerance?

Reply:
The speed tolerance in production code is 50%, here is to align the test case with it.